src/hexEditorProvider.ts

```diff
@@ -171,7 +171,7 @@ export class HexEditorProvider implements vscode.CustomEditorProvider<HexDocumen
 	private getHtmlForWebview(webview: vscode.Webview): string {
 		// Convert the styles and scripts for the webview into webview URIs
 		const scriptUri = webview.asWebviewUri(vscode.Uri.joinPath(this._context.extensionUri, "dist", "editor.js"));
-		const styleUri = webview.asWebviewUri(vscode.Uri.joinPath(this._context.extensionUri, "dist", "hexEdit.css"));
+		const styleUri = webview.asWebviewUri(vscode.Uri.joinPath(this._context.extensionUri, "dist", "editor.css"));
```

Comment:
Does editor.css get built from the inlined styles? How does this file come to be

Reply:
Linaria has an esbuild plugin which generates `editor.css` automatically from the styles in JavaScript. Normally I would use styled-components or css modules for this, but this aren't available in esbuild. I'm actually liking Linaria quite a bit, it has no runtime overhead since the styles get compiled to CSS and replaced with just classnames in the source code.

For example this is what editor.css looks like now:

![image](https://user-images.githubusercontent.com/2230985/142040534-7301d53e-a693-412e-b501-9ec4eed35df0.png)